pages/[chain]/[address].tsx

```diff
@@ -366,7 +366,9 @@ const SynthPage: React.FC<Props> = ({ data, chainId, relatedSynths }) => {
       <MainWrapper>
         <div>
           <About description={data.description} />
-          <Information synth={freshData as Synth<{ type: ContractType }>} />
+          {freshData && (
```

Comment:
i added this fresh data check because i was getting random crashes locally due to this being undefined on first load. might not be a problem in production since it uses infura and not a local hardhat node, but doesnt hurt to have this check